utils/remote.cc

```diff
@@ -2457,21 +2406,50 @@ void tr_curl_easy_cleanup(CURL* curl)
     }
 }
 
-int flush(char const* rpcurl, tr_variant* benc, RemoteConfig& config)
+[[nodiscard]] long get_timeout_secs(std::string_view req)
+{
+    if (req.find(R"("method":"blocklist-update")") != std::string_view::npos)
+    {
+        return 300L;
+    }
+
+    return 60L; /* default value */
+}
+
+[[nodiscard]] std::string serialize_payload(tr_variant const* const var, RemoteConfig const& config)
+{
+    auto serde = tr_variant_serde::json();
+    serde.compact();
+    fmt::print(stderr, "{:s}:{:d} serializing payload to {:d}\n", __FILE__, __LINE__, static_cast<int>(config.network_style));
```

Comment:
I *think* this is what you meant.

```suggestion
    if (config.debug)
        fmt::print(stderr, "{:s}:{:d} serializing payload to {:d}\n", __FILE__, __LINE__, static_cast<int>(config.network_style));
    }
```